src/base/static/components/input-form/index.js

```diff
@@ -81,23 +101,22 @@ class InputForm extends Component {
     }));
   }
 
-  onFieldChange(fieldName, fieldValue, isValid, message) {
+  onGeometryStyleChange(style) {
+    this.geometryStyle = style;
+  }
+
+  onFieldChange(fieldName, fieldValue, isValid) {
     const newState = this.state.fields
```

Comment:
`newState` isn't a very good variable name. Maybe something like `updatedFieldConfig` would be better?